test/bdd/fixtures/vault-server/docker-compose.yml

```diff
@@ -42,8 +42,7 @@ services:
       - KMS_ZCAP_ENABLE=true
       - KMS_TLS_SYSTEMCERTPOOL=true
       - KMS_TLS_CACERTS=/etc/tls/ec-cacert.pem
-      - KMS_TLS_SERVE_CERT=/etc/tls/ec-pubCert.pem
```

Comment:
Any specific reason to remove TLS from KMS setup ?

Reply:
I will rollback it